addon.xml

```diff
@@ -12,11 +12,13 @@
 		<forum>https://forum.kodi.tv/showthread.php?tid=319709</forum>
 		<summary lang="en_GB">A chromecast-like screensaver for Kodi</summary>
 		<summary lang="pt_PT">Um screensaver idêntico ao screensaver original chromecast</summary>
+		<summary lang="es_SP">Un protector de pantalla tipo Chromecast para Kodi</summary>
 		<description lang="en_GB">Display beautiful pictures originally from the chromecast screensaver. You can also display your own photos along with its respective information.</description>
 		<description lang="pt_PT">Apresenta imagens lindíssimas originalmente pertencentes ao screensaver do chromecast. Pode também apresentar as suas próprias imagens conjuntamente com a respectiva informação.</description>
+		<description lang="es_SP">Muestra bellas imágenes como las del protector de pantalla de chromecast. También puede mostrar sus propias fotos junto con su respectiva información.</description>
 		<news>v1.0.2 [CR]
 			- Hide logos. [CR]
-			-animations
+			- Animations
```

Comment:
Please drop this change as it refers to changes between versions. It will be changed in the next update